internal/data_generator/internal/base_structs.go

```diff
@@ -79,7 +79,7 @@ func (es ${structName}) Resize(from, to int) {
 	*es.orig = (*es.orig)[from:to]
 }`
 
-const messageTemplate = `// ${structName} ${description}.
+const messageTemplate = `${description}
```

Comment:
Ah it seems some other PRs after this one needs this same change.

Reply:
:)